api/src/main/java/org/openmrs/module/fhir/api/diagnosticreport/handler/LaboratoryHandler.java

```diff
@@ -27,20 +27,47 @@ public LaboratoryHandler() {
 		super();
 	}
 
-    private static final String ServiceCategory = "LAB";
+	private static final String ServiceCategory = "LAB";
 
-    public String getServiceCategory() {
-        return ServiceCategory;
-    }
+	public String getServiceCategory() {
+		return ServiceCategory;
+	}
+
+	@Override
+	public DiagnosticReport getFHIRDiagnosticReport(Encounter omrsiagnosticReport) {
```

Comment:
I think this should receive a UUID, and not an encounter. Only the Handler itself knows that the UUID is for an encounter....